monitoring/grafana/notifiers/notifiers.yml

```diff
@@ -0,0 +1,23 @@
+notifiers:
+  - name: discord-alerts
+    type: discord
+    uid: discord
+    org_name: Main Org.
+    is_default: true
+    secure_settings:
+      url: $LP_DISCORD_WEBHOOK
+  - name: pagerduty-alerts
```

Comment:
So for our other alerts like prometheus or papertrail, we send a single webhook to pagerduty and then pagerduty itself integrates with discord and sends the message there. Is there an advantage of sending it directly in case of grafana? I know that on Slack it sends a nicely formatted message with some useful information about the trigger condition of the alert as well, do they have good support for discord as well? 🙌 

Also, are you still configuring pagerduty to send a msg on the channel as well, or only grafana? Not sure which I prefer (no link to pager duty incident vs double notification every time)

Reply:
yeah i'll keep discord there but not activate it by default. it'll come in handy for non pagable metrics we wanna notify about